client/lib/signup/step-actions.js

```diff
@@ -24,6 +24,7 @@ import { getDesignType } from 'state/signup/steps/design-type/selectors';
 import { getSiteTitle } from 'state/signup/steps/site-title/selectors';
 import { getSurveyVertical, getSurveySiteType } from 'state/signup/steps/survey/selectors';
 import { getSiteType } from 'state/signup/steps/site-type/selectors';
+import getSignupStepsSiteTopic from 'state/selectors/get-signup-steps-site-topic';
```

Comment:
I missed this in #28164, but can we move this selector (and its test) to `state/signup/steps/site-topic/selectors`, just to follow the previous Signup redux conventions (and declutter `state/selectors/`?

You can handle it in a separate PR if you'd like.

Reply:
Moved in 3c92d33.